simulator/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM rust:1.70.0-bullseye
```

Comment:
Out of interest - any reason to use `bullseye` rather than `bullseye-slim` here? Alignment across the stack is a fair reason.

Reply:
no reason at all. will try slim

Reply:
slim seems to working just fine. I would rather use slim to keep the image smaller. let me know if you prefer to go back to bullseye (no slim) to keep consistency across the stack